src/components/CompareTable/CompareCompetencesTable.tsx

```diff
@@ -0,0 +1,43 @@
+import { components } from '@/api/schema';
+import { getLocalizedText } from '@/utils';
+import { useTranslation } from 'react-i18next';
+import { CompareCompetencesTableRow, CompareCompetencesTableRowData } from './CompareCompetencesTableRow';
+
+interface CompareCompetencesTableProps {
+  rows: CompareCompetencesTableRowData[];
+  opportunityName?: components['schemas']['LokalisoituTeksti'];
+  className: string;
+}
+
+export const CompareCompetencesTable = ({ rows, opportunityName, className }: CompareCompetencesTableProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className={`overflow-x-auto ${className}`.trim()}>
+      <table className="w-full" border={0} cellPadding={0} cellSpacing={0}>
+        <thead className="after:content-[''] after:block after:h-5">
+          <tr className="font-normal text-body-md border-b border-inactive-gray">
```

Comment:
`font-normal` is not needed as the `text-body-md` already has the same `font-weight`.